docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/AveragedPerceptron.cs

```diff
@@ -111,11 +129,16 @@ private static void PrintMetrics(BinaryClassificationMetrics metrics)
             Console.WriteLine($"Accuracy: {metrics.Accuracy:F2}");
             Console.WriteLine($"AUC: {metrics.AreaUnderRocCurve:F2}");
             Console.WriteLine($"F1 Score: {metrics.F1Score:F2}");
-            Console.WriteLine($"Negative Precision: {metrics.NegativePrecision:F2}");
+            Console.WriteLine($"Negative Precision: {metrics.NegativePrecision:F2}")
+                ;
```

Comment:
I don't think this is the best formatting. Maybe do it like this:

Console.WriteLine($"Negative Precision: " +
    $"{metrics.NegativePrecision:F2}");